### PR TITLE
[Term Entry] PHP Math Function: hypot()

### PR DESCRIPTION
**Pull request by chris-larham-1983**
### Description

- Added the hypot() method for PHP
- Closes Issue #2272 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by chris-larham-1983** [comment]
@KTom101 Thanks for assigning me this Issue, I look forward to your feedback on this solution 🙂

File: content/php/concepts/math-functions/terms/hypot/hypot.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+The function accepts two `float` inputs and returns a value of type `float`.
+
+## Syntax
+
+```pseudo
+$hypotenuse = hypot(length1, length2);
+```
+
+Where `length1` and `length2` are the two lengths of a right-angle triangle.
```
[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/php/concepts/math-functions/terms/hypot/hypot.md:27]: html cannot be used for formatting in docs

File: content/php/concepts/math-functions/terms/hypot/hypot.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  - 'Geometry'
+  - 'Math'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`hypot()`** function returns the length of the hypotenuse of a right-angle triangle. Given a right-angle triangle with sides `$length1` and `$length2`, it returns the square root of (`$length1`<sup>2</sup> + `$length2`<sup>2</sup>).
+
+The function accepts two `float` inputs and returns a value of type `float`.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/hypot/hypot.md:19]: I would recommend moving this after line 27

File: content/php/concepts/math-functions/terms/hypot/hypot.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+This example results in the following output:
+
+```shell
+10
+```
+
+## Codebyte Example
+
+This example returns the hypotenuse for two right-angle triangles - one with sides whose lengths are expressed in whole numbers, and one with sides whose lengths contain a fractional component:
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/hypot/hypot.md:48]: This example is largely redundant. Codebytes are optional. Please revise this example to demonstrate some alternative functionality or delete it.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@chris-larham-1983 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

File: content/php/concepts/math-functions/terms/hypot/hypot.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
+---
+Title: 'hypot()'
+Description: 'Returns the length of the hypotenuse of a right-angle triangle.'
```
[the remark this answers is not in the document]
Comment (chris-larham-1983) [reply content/php/concepts/math-functions/terms/hypot/hypot.md]: @caupolicandiaz I have implemented this change in the most recent commit.

File: content/php/concepts/math-functions/terms/hypot/hypot.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Geometry'
+  - 'Math'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`hypot()`** function returns the length of the hypotenuse of a right-angle triangle. Given a right-angle triangle with sides `$length1` and `$length2`, it returns the square root of (`$length1`<sup>2</sup> + `$length2`<sup>2</sup>).
```
[the remark this answers is not in the document]
Comment (chris-larham-1983) [reply content/php/concepts/math-functions/terms/hypot/hypot.md]: @caupolicandiaz I have implemented this change in the most recent commit.

File: content/php/concepts/math-functions/terms/hypot/hypot.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (chris-larham-1983) [reply content/php/concepts/math-functions/terms/hypot/hypot.md]: @caupolicandiaz I have implemented this change in the most recent commit.

File: content/php/concepts/math-functions/terms/hypot/hypot.md (hunk 2 above)
Comment (chris-larham-1983) [reply content/php/concepts/math-functions/terms/hypot/hypot.md]: @caupolicandiaz I have implemented this change in the most recent commit.

File: content/php/concepts/math-functions/terms/hypot/hypot.md (hunk 3 above)
Comment (chris-larham-1983) [reply content/php/concepts/math-functions/terms/hypot/hypot.md]: @caupolicandiaz I have removed the comparison of two 90-degree triangles, and just left one example that Docs viewers can run to ensure the function works as expected (a 90-degree triangle with sides of length `12` and `16`).  Personally, I like to run a Codebyte snippet to ensure that I have understood the concept correctly.  I am happy to totally remove this Codebyte if you would prefer.

**Comment by chris-larham-1983** [comment]
@caupolicandiaz I have made the changes you suggested in the most recent commit 🙂

**Review by caupolicandiaz — APPROVED** [review]
@chris-larham-1983 Looks good. I just pushed a few small changes to improve the syntax section. This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@chris-larham-1983 Looks good.

**Comment by SSwiniarski** [comment]
@chris-larham-1983 Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/math-functions/hypot

image

**Pull request merged**